ubuntu-advantage

```diff
@@ -82,6 +34,7 @@ enable_esm() {
 disable_esm() {
     if [ -f "$REPO_LIST" ]; then
         mv "$REPO_LIST" "${REPO_LIST}.save"
+        rm  "$APT_KEYS_DIR/$REPO_KEY_FILE"
```

Comment:
This `rm` should be guarded with `-f`, or the script can fail if that file doesn't exist by any chance:

```
ubuntu@precise-esm:~$ sudo rm /etc/apt/trusted.gpg.d//ubuntu-keyring-esm.gpg
ubuntu@precise-esm:~$ sudo ubuntu-advantage disable-esm
rm: cannot remove `/etc/apt/trusted.gpg.d/ubuntu-keyring-esm.gpg': No such file or directory
ubuntu@precise-esm:~$ 
```